cd/helm/nef/templates/mongo-express-nef-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f docker-compose-resolved.yaml -o nef
+    kompose.version: 1.26.1 (HEAD)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: mongo-express-nef
+  name: mongo-express-nef
+spec:
+  ports:
+    - name: "8081"
```

Comment:
Change Name self-generated 8080 makes some problems at route yaml of service because it will be understand like number not like name. Maybe we can set something like <service_name>-port